ironfish/src/rpc/routes/wallet/utils.ts

```diff
@@ -104,6 +107,30 @@ export function deserializeRpcAccountImport(accountImport: RpcAccountImport): Ac
           sequence: accountImport.createdAt.sequence,
         }
       : null,
+    multiSigKeys: accountImport.multiSigKeys
+      ? deserializeRpcAccountMultiSigKeys(accountImport.multiSigKeys)
+      : undefined,
+  }
+}
+
+export function deserializeRpcAccountMultiSigKeys(
+  rpcMultiSigKeys: RpcMultiSigKeys,
+): MultiSigKeys | undefined {
+  if (!rpcMultiSigKeys) {
+    return undefined
+  }
+
+  if (isSignerMultiSig(rpcMultiSigKeys)) {
```

Comment:
Should we do any validation in these methods to make sure if keyPackge is present identifier is too?

Reply:
Yeah, updated this function: https://github.com/iron-fish/ironfish/pull/4652/files#diff-14efb6c75810d0e95c33ceaca29faebf9473a5af44f3936a1711831c35c2eb58R78-R84